src/applications/personalization/dashboard/tests/components/claims-and-appeals/ClaimsAndAppealsLighthouse.unit.spec.jsx

```diff
@@ -151,13 +148,10 @@ describe('ClaimsAndAppeals component', () => {
             claimsAvailability: claimsAvailability.UNAVAILABLE,
           },
         };
-        view = renderInReduxProvider(
-          <ClaimsAndAppeals dataLoadingDisabled useLighthouseClaims />,
-          {
-            initialState,
-            reducers,
-          },
-        );
+        view = renderInReduxProvider(<ClaimsAndAppeals dataLoadingDisabled />, {
+          initialState,
+          reducers,
+        });
```

Comment:
As a side note, this `dataLoadingDisabled` feels like a code smell to me. Maybe in the future we can look at what we could do to not have this required for testing.